datascience-notebook/Dockerfile

```diff
@@ -4,23 +4,27 @@ LABEL maintainer=analytics-platform-tech@digital.justice.gov.uk
 
 USER root
 
-ENV PATH=$PATH:$HOME/.local/bin
-ENV NB_UID=1001
-ENV CHOWN_HOME=no
+ENV PATH=$PATH:$HOME/.local/bin \
+    NB_UID=1001 \
+    CHOWN_HOME=no
 
-COPY ./files/apt_packages /tmp/
-
-RUN conda install boto3 black flake8 jupyterlab_code_formatter \
-    && apt-get update \
-    && apt-get install -y $(cat /tmp/apt_packages) \
+RUN apt-get update \
+    && apt-get install -y \
+    ca-certificates-java \
+    openjdk-8-jdk \
+    openssh-client \
+    software-properties-common \
+    gdal-bin \
+    libspatialindex-dev \
     && rm -rf /var/lib/apt/lists/*
 
+RUN pip install --upgrade boto3 black flake8 jupyterlab_code_formatter nbstripout
```

Comment:
The switch from pip to conda makes me uneasy. I suggest we do try it, but how about doing it in a separate PR, so it is considered separately?

Reply:
Yeah I'm trying to switch back now. I'm actually getting build failures purely down to Conda wanting python 3.7 and 3.6 is installed 🤷